### PR TITLE
add lifcl (lattice crosslink-nx) eval board definition file

### PR DESCRIPTION
**Pull request by slagernate**
tested on crosslink-nx board, blinking all 14 LEDs

### CODE REVIEW

File: amaranth_boards/ecp5_5g_evn.py [hunk 1]
```diff
@@ -55,9 +55,9 @@ def bank6_iostandard(self):
                       attrs=Attrs(IO_TYPE=bank1_iostandard)),
         *ButtonResources(pins="P4", invert=True,
                          attrs=Attrs(IO_TYPE=bank6_iostandard)),
-        *SwitchResources(pins={1: "J1", 2: "H1", 3: "K1"}, invert=True,
+        *SwitchResources(pins={0: "J1", 1: "H1", 2: "K1"}, invert=True,
                          attrs=Attrs(IO_TYPE=bank6_iostandard)),
-        *SwitchResources(pins={4: "E15", 5: "D16", 6: "B16", 7: "C16", 8: "A16"}, invert=True,
+        *SwitchResources(pins={3: "E15", 4: "D16", 5: "B16", 6: "C16", 7: "A16"}, invert=True,
```
Comment (whitequark) [inline amaranth_boards/ecp5_5g_evn.py:60]: Unrelated change?

**Review by whitequark — COMMENTED** [review]
Which board specifically is it? LIFCL-VVML-EVN?

File: amaranth_boards/ecp5_5g_evn.py (hunk 1 above)
Comment (slagernate) [reply amaranth_boards/ecp5_5g_evn.py:60]: re: board name --this is the most vanilla crosslink-nx eval board (they call it the "CrossLink-NX Evaluation Board"). Can possibly keep this as lifcl_evn.py, then can use lifcl_vip_evn.py, lifcl_vvml_evn.py, lifcl-33_vvml_evn.py for the other crosslinknx boards. 

re: ecp5 changes --it seems resources cannot be found if they skip index 0. I.e. I tried using the same format for LEDResources() that the ecp5 platform file was using (1, 2, 3, ...), but Blinky() couldn't find any LEDs (now using (0, 1, 2, ...). Perhaps something changed, or they never tested blinking of the LEDs on the ecp5 board.

**Comment by whitequark** [comment]
I... think you unintentionally deleted the repository with the PR?
